src/local-fs.ts

```diff
@@ -10,7 +10,7 @@ import { unlockFile, readFile } from '@verdaccio/file-locking';
 import { Callback, Logger, Package, ILocalPackageManager, IUploadTarball } from '@verdaccio/types';
 import { getCode, getInternalError, getNotFound, VerdaccioError } from '@verdaccio/commons-api/lib';
 
-export const fileExist = 'EEXISTS';
+export const fileExist = 'EEXIST';
```

Comment:
Why this change?
If we modify that breaks the project here https://github.com/verdaccio/verdaccio/blob/master/src/lib/constants.ts#L161 since I am expecting such thing. It is not ideal to have suck weak relationship, but, I wonder why this change.

Reply:
Ok, here some context. The keyword `EEXISTS` a legacy from sinopia, it seems at Node.js `0.10` the codes were not documented and the first reference I've found by far is at Node.js 4.x https://nodejs.org/docs/latest-v4.x/api/errors.html , but this error handler is unrelated with the native one `EEXIST (File exists): An existing file was the target of an operation that required that the target not exist.`

At Verdaccio still we rely on `EEXISTS` so, this is super misleading, totally agree, but, we will change this in a future PR to avoid confusions

Reply:
I changed the constant name back. I had renamed the constant so I could reuse it in a check for that error, because it seemed strange to have to switch between error codes. Since the constant was exported I assumed any code that relied on that constant would import it and so I hadn't considered that it might cause issues.